<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Broadcaster - UDP output stream creation - Encryption AES 128"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="21.011">
  <properties />
            <testcase name="broadcasterPullInCreation" time="21.011" classname="com.zixi.testing.BroadcasterUDPOutputStreamCreationTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.zixi.ssh.SshJcraftClient.callCommand(SshJcraftClient.java:86)
  com.zixi.ssh.SshJcraftClient.callCommand(SshJcraftClient.java:129)
  com.zixi.tools.BroadcasterLoggableApiWorker.getPid(BroadcasterLoggableApiWorker.java:37)
  com.zixi.testing.BroadcasterUDPOutputStreamCreationTest.broadcasterPullInCreation(BroadcasterUDPOutputStreamCreationTest.java:25)
  sun.reflect.GeneratedMethodAccessor12.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:771)
  org.testng.TestRunner.run(TestRunner.java:621)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1032)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.Surefire.run(Surefire.java:180)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
  org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
